sdk/storage/inc/blobs/blob_client.hpp

```diff
@@ -192,7 +181,7 @@ namespace Azure { namespace Storage { namespace Blobs {
      * parameters to execute this function.
      * @return A SetAccessTierResponse on successfully setting the tier.
```

Comment:
Not consistent with the actual return type.

Reply:
fixed